customer-config/checkout-config/che.plulist.v2.json

```diff
@@ -100,15 +100,15 @@
       "description": "This is an entry for a specific item, and is placed inside a PLU list.",
       "type": "object",
       "required": [
-        "identifier",
+        "id",
```

Comment:
Important notice that the id points to the item identifier as described in the description.